ls1intum/Artemis
# `Development`: Add Docker setup for documentation previews

### Checklist
#### General

### Motivation and Context


I didn't know that [readthedocs](readthedocs.org/projects/artemis-platform) also produces builds for PRs.
Also, I wanted to have a preview before committing bigger changes to the documentation.
Of course, it's nicer to not have to install it locally and therefore Docker seemed to be a viable option.

### Description

Added documentation which hopefully points more to  the variety of builds on the read the docs platform.

Added documentation and a Dockerfile to easily get a locally docker-hosted autobuild preview setup for the docs.

### Steps for Testing

Prerequisites:
- 1 Developer

1. run the commands mentioned in the [documentation](https://artemis-platform--5647.org.readthedocs.build/en/5647/)
2. check if you can edit and preview as desired in your normal developer documentation workflow

### Review Progress





#### Code Review
#### Manual Tests
